src/upload/upload_bg.py

```diff
@@ -0,0 +1,103 @@
+"""Blender script to create blend file to upload."""
+import json
+import logging
+import os
+import sys
+from importlib import import_module
+
+import bpy
+
+FILENAME = sys.argv[-1]
+HANA3D_NAME = sys.argv[-2]
+HANA3D_EXPORT_DATA = sys.argv[-3]
+
+module = import_module(HANA3D_NAME)
```

Comment:
Algum motivo para usar o `import_module` ao invés de usar imports relativos? Isso é por ser um script do blender?

Reply:
Import relativo não funciona porque ele é rodado como um script em um Blender chamado em um subprocess.
Aqui tem uma explicação detalhada:
https://stackoverflow.com/questions/14132789/relative-imports-for-the-billionth-time